src/collective/solr/profiles/default/registry.xml

```diff
@@ -11,7 +11,7 @@
     <value key="max_results">10000000</value>
     <value key="exclude_user">False
     </value>
-    <value key="search_pattern">+(Title:{value}^5 OR Description:{value}^2 OR SearchableText:{value} OR SearchableText:({base_value}) OR searchwords:({base_value})^1000) +showinsearch:True
+    <value key="search_pattern">+(Title:{value}^5 OR Description:{value}^2 OR SearchableText:{value} OR SearchableText:({base_value}) OR searchwords:({base_value})^1000) -showinsearch:False
```

Comment:
@reinhardt could you elaborate why you changed "showinsearch" here?

Reply:
A new/custom content type may not have the ISolrFields behavior activated. That means the value for "showinsearch" will be missing. If we require it to be True, then this content type will not be searchable with solr. That's probably unexpected, and also a change from before these fields were converted to a behavior. The original way was that showinsearch was True by default, so all instances of new content types would show up in the search unless you explicitly toggled showinsearch. Searching for content that has showinsearch not set to False, i.e. explicitly True or missing altogether, comes pretty close to that I think.